albam/engines/mtfw/material.py

```diff
@@ -1013,103 +1021,176 @@ def _has_mtfw_shader_group(bl_mat):
-    unk_flag_48: bpy.props.BoolProperty(default=0)
-
-    unk_01: bpy.props.IntProperty(default=0)
-    unk_02: bpy.props.IntProperty(default=0)
-    unk_03: bpy.props.IntProperty(default=0)
-    unk_04: bpy.props.IntProperty(default=0)
-    unk_05: bpy.props.IntProperty(default=0)
-    unk_06: bpy.props.IntProperty(default=0)
-    unk_07: bpy.props.IntProperty(default=0)
-    unk_08: bpy.props.IntProperty(default=0)
-    unk_09: bpy.props.IntProperty(default=0)
-    unk_param_01: bpy.props.FloatProperty(default=0.0)
-    unk_param_02: bpy.props.FloatProperty(default=0.0)
-    unk_param_03: bpy.props.FloatProperty(default=0.0)
-    unk_param_04: bpy.props.FloatProperty(default=0.0)
-    unk_param_05: bpy.props.FloatProperty(default=0.0)
-    cubemap_roughness: bpy.props.FloatProperty(default=0.0)
-    unk_param_07: bpy.props.FloatProperty(default=0.0)
-    unk_param_08: bpy.props.FloatProperty(default=0.0)
-    unk_param_09: bpy.props.FloatProperty(default=0.0)
-    unk_param_10: bpy.props.FloatProperty(default=0.0)
-    detail_normal_power: bpy.props.FloatProperty(default=0.0)
-    detail_normal_multiplier: bpy.props.FloatProperty(default=0.0)
-    unk_param_13: bpy.props.FloatProperty(default=0.0)
-    unk_param_14: bpy.props.FloatProperty(default=0.0)
-    unk_param_15: bpy.props.FloatProperty(default=0.0)
-    unk_param_16: bpy.props.FloatProperty(default=0.0)
-    unk_param_17: bpy.props.FloatProperty(default=0.0)
-    unk_param_18: bpy.props.FloatProperty(default=0.0)
-    unk_param_19: bpy.props.FloatProperty(default=0.0)
-    unk_param_20: bpy.props.FloatProperty(default=0.0)
-    normal_scale: bpy.props.FloatProperty(default=0.0)
-    unk_param_22: bpy.props.FloatProperty(default=0.0)
-    unk_param_23: bpy.props.FloatProperty(default=0.0)
-    unk_param_24: bpy.props.FloatProperty(default=0.0)
-    unk_param_25: bpy.props.FloatProperty(default=0.0)
-    unk_param_26: bpy.props.FloatProperty(default=0.0)
+    attr_enum = bpy.props.EnumProperty(
+        name="Attribute",
+        description="Select surface attribute",
+        items=[
+            ("0x1", "ATTR_BRIDGE", "", 1),
+            ("0x8", "ATTR_NUKI", "", 2),
+            ("0x10", "ATTR_SOLID", "", 3),
+            ("0x20", "ATTR_OVERLAP", "", 4),
+            ("0x40", "ATTR_TRANSPARENT", "", 5),
+        ],
+        default="0x10",
+        options=set()
+    )
+    vtype_enum = bpy.props.EnumProperty(
+        name="VTYPE",
+        description="Select vertex type",
+        items=[
+            ("0x0", "VTYPE_SKIN", "Skinned mesh with up to 4wt", 1),
+            ("0x1", "VTYPE_SKINEX", "Skinned mesh extended", 2),
+            ("0x2", "VTYPE_NONSKIN", "Static mesh", 3),
+            ("0x3", "VTYPE_NONSKIN_COL", "Static mesh with vertex colors", 4),
+            ("0x4", "VTYPE_SHAPE", "", 5),
+            ("0x5", "VTYPE_SKIN_COL", "", 6),
+        ],
+        default="0x0",
+        options=set()
+    )
+    func_skin_enum = bpy.props.EnumProperty(
```

Comment:
Would be so nice to have a button or something to auto-select it based on the actual total influences. 
For later.